CHANGELOG.md

```diff
@@ -98,6 +98,13 @@ All notable changes to this project will be documented in this file.
         * Methods removed
             * `get_frequency_response()` [#823](https://github.com/ni/nimi-python/issues/823)
             * `export_signal()` - [#828](https://github.com/ni/nimi-python/issues/828)
+* ### NI-ModInst
+    * #### Added
+    * #### Changed
+        * Indexing on the modinst session is no longer allowed
```

Comment:
modinst session should be `nimodinst.Session` (proper Python class name, no ambiguity)

Reply:
Changed